compiler/tools/snax_opt_main.py

```diff
@@ -46,7 +46,7 @@ def __init__(
 
         ## Add custom dialects & passes
         # FIXME: override upstream accfg dialect. Remove this after upstreaming full downstream accfg dialect.
-        del self.ctx._registered_dialects["accfg"]  # pyright: ignore
+        self.ctx._registered_dialects.pop("accfg", None)
```

Comment:
This change is so that if an xdsl is used that does *not* have an `accfg` dialect, we don't get an error saying "hurr durr no "accfg" in `_registered_dialects`....